fabtests/prov/efa/src/rdm_rnr_queue_resend.c

```diff
@@ -140,7 +141,8 @@ static int trigger_rnr_queue_resend(enum fi_op atomic_op, void *result, void *co
 		for (i = 0; i < global_expected_rnr_error; i++) {
 			switch (opts.rma_op) {
 			case FT_RMA_WRITE:
-				ret = ft_post_rma(FT_RMA_WRITE, tx_buf, opts.transfer_size,
+			case FT_RMA_WRITEDATA:
```

Comment:
does this test need firmware support rdma write to trigger the error? does it also work for emulated write path?

Reply:
The CI isn't running on p5, so it works for emulated writedata

Reply:
I also ran the test with `FI_EFA_USE_DEVICE_RDMA=0` on p5 and it passed